tests/acceptance/features/apiSharingNg/mountOrUnmountShare.feature

```diff
@@ -54,3 +54,35 @@ Feature:  mount or unmount incoming share
       | resource      |
       | textfile0.txt |
       | FolderToShare |
+
+
+  Scenario Outline: mount shared resource when auto-sync is disabled
+    Given user "Brian" has disabled auto-accepting
```

Comment:
```suggestion
    Given user "Brian" has disabled the auto-sync share
```
The step implementation is the same but we used the auto-accept term in sharing API.